app/views/mentor/analyses/index.html.haml

```diff
@@ -1,7 +1,9 @@
 #mentor-analyses
   .lo-container
     =form_tag({}, method: :get) do
-      =select_tag :track_id, options_for_select(@tracks.map{|t|[t.title, t.id]}, @track_id), include_blank: true
+      =select_tag :track_id, options_for_select(@tracks.map{|t|[t.title, t.id]}, @track_id), include_blank: "Track"
+      =select_tag :ops_status, options_for_select(IterationAnalysis.distinct.pluck(:ops_status), @ops_status), include_blank: "Ops status"
+      =select_tag :analysis_status, options_for_select(IterationAnalysis.distinct.pluck(:analysis_status).compact, @analysis_status), include_blank: "Analysis Status"
```

Comment:
You need `.compact` here due to the fact that the `analysis_status` column is nullable, right?

Reply:
Correct, yep. In my local testing one of the values was `nil`, which made me realise, thankfully.